apps/wing/package.json

```diff
@@ -29,22 +29,23 @@
     "unbump": "node unversion.js"
   },
   "dependencies": {
+    "@winglang/sdk": "file:../../libs/wingsdk",
     "chalk": "^4.1.2",
     "codespan-wasm": "0.3.0",
-    "commander": "^10.0.0",
+    "commander": "^8.0.0",
```

Comment:
I'm not sure we're okay with downgrading our dependency so much just to satisfy the peer dependency of @fig/complete-commander. Looks like it's all contained in this single file https://github.com/withfig/autocomplete-tools/blob/main/integrations/commander/src/index.ts. Maybe we can just copy/paste with attribution instead, assuming it still works with the latest version of commander.

Reply:
True, I had also mixed feelings about this, will try to tweak this with commander 10 instead